<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
       <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>水印添加与检测</string>
  </property>
  <widget class="QWidget" name="centralwidget">
      <widget class="QProgressBar" name="progressBar">
          <property name="geometry">
              <rect>
                  <x>30</x>
                  <y>370</y>
                  <width>681</width>
                  <height>23</height>
              </rect>
          </property>
          <property name="value">
              <number>0</number>
          </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
              <rect>
                  <x>20</x>
                  <y>10</y>
                  <width>691</width>
                  <height>351</height>
              </rect>
          </property>
          <widget class="QGroupBox" name="groupBox">
              <property name="geometry">
                  <rect>
                      <x>10</x>
                      <y>10</y>
                      <width>671</width>
                      <height>71</height>
                  </rect>
              </property>
              <property name="title">
                  <string>选择与连接</string>
              </property>
              <property name="flat">
                  <bool>false</bool>
              </property>
              <property name="checkable">
                  <bool>false</bool>
              </property>
              <property name="checked">
                  <bool>false</bool>
              </property>
              <widget class="QComboBox" name="tb_comboBox">
                  <property name="geometry">
                      <rect>
                          <x>350</x>
                          <y>30</y>
                          <width>111</width>
                          <height>22</height>
                      </rect>
                  </property>
                  <item>
                      <property name="text">
                          <string>请先选择数据库</string>
                      </property>
                  </item>
              </widget>
              <widget class="QPushButton" name="login_pushButton">
                  <property name="geometry">
                      <rect>
                          <x>10</x>
                          <y>30</y>
                          <width>71</width>
                          <height>23</height>
                      </rect>
                  </property>
                  <property name="toolTip">
                      <string>点击登陆MySQL</string>
                  </property>
                  <property name="text">
                      <string>登录MySQL</string>
                  </property>
              </widget>
              <widget class="QComboBox" name="db_comboBox">
                  <property name="geometry">
                      <rect>
                          <x>160</x>
                          <y>30</y>
                          <width>105</width>
                          <height>20</height>
                      </rect>
                  </property>
                  <property name="insertPolicy">
                      <enum>QComboBox::InsertAtBottom</enum>
                  </property>
                  <property name="sizeAdjustPolicy">
                      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
                  </property>
                  <item>
                      <property name="text">
                          <string>请先连接MySQL</string>
                      </property>
                  </item>
              </widget>
              <widget class="QLabel" name="label">
                  <property name="geometry">
                      <rect>
                          <x>110</x>
                          <y>33</y>
                          <width>51</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>数据库：</string>
                  </property>
              </widget>
              <widget class="QLabel" name="label_2">
                  <property name="geometry">
                      <rect>
                          <x>300</x>
                          <y>33</y>
                          <width>51</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>数据表：</string>
                  </property>
              </widget>
              <widget class="QLabel" name="label_3">
                  <property name="geometry">
                      <rect>
                          <x>495</x>
                          <y>33</y>
                          <width>31</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>密钥：</string>
                  </property>
              </widget>
              <widget class="QLineEdit" name="key_lineEdit">
                  <property name="geometry">
                      <rect>
                          <x>530</x>
                          <y>30</y>
                          <width>81</width>
                          <height>20</height>
                      </rect>
                  </property>
                  <property name="readOnly">
                      <bool>true</bool>
                  </property>
              </widget>
              <widget class="QPushButton" name="key_pushButton">
                  <property name="geometry">
                      <rect>
                          <x>622</x>
                          <y>29</y>
                          <width>41</width>
                          <height>23</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>…</string>
                  </property>
              </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
              <property name="geometry">
                  <rect>
                      <x>10</x>
                      <y>100</y>
                      <width>671</width>
                      <height>61</height>
                  </rect>
              </property>
              <property name="title">
                  <string>参数设置</string>
              </property>
              <widget class="QLabel" name="label_6">
                  <property name="geometry">
                      <rect>
                          <x>20</x>
                          <y>30</y>
                          <width>101</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>水印比例系数：1/</string>
                  </property>
              </widget>
              <widget class="QLineEdit" name="scale_lineEdit">
                  <property name="geometry">
                      <rect>
                          <x>121</x>
                          <y>28</y>
                          <width>51</width>
                          <height>20</height>
                      </rect>
                  </property>
              </widget>
              <widget class="QLabel" name="label_7">
                  <property name="geometry">
                      <rect>
                          <x>250</x>
                          <y>30</y>
                          <width>61</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>属性个数：</string>
                  </property>
              </widget>
              <widget class="QLineEdit" name="attrNum_lineEdit">
                  <property name="geometry">
                      <rect>
                          <x>313</x>
                          <y>28</y>
                          <width>61</width>
                          <height>20</height>
                      </rect>
                  </property>
              </widget>
              <widget class="QLabel" name="label_8">
                  <property name="geometry">
                      <rect>
                          <x>430</x>
                          <y>30</y>
                          <width>81</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>不重要比特数：</string>
                  </property>
              </widget>
              <widget class="QLineEdit" name="lsb_lineEdit">
                  <property name="geometry">
                      <rect>
                          <x>520</x>
                          <y>27</y>
                          <width>61</width>
                          <height>20</height>
                      </rect>
                  </property>
              </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
              <property name="geometry">
                  <rect>
                      <x>10</x>
                      <y>180</y>
                      <width>671</width>
                      <height>80</height>
                  </rect>
              </property>
              <property name="title">
                  <string>水印添加</string>
              </property>
              <widget class="QPushButton" name="add_pushButton">
                  <property name="geometry">
                      <rect>
                          <x>260</x>
                          <y>30</y>
                          <width>75</width>
                          <height>23</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>添加</string>
                  </property>
              </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
              <property name="geometry">
                  <rect>
                      <x>10</x>
                      <y>270</y>
                      <width>671</width>
                      <height>71</height>
                  </rect>
              </property>
              <property name="title">
                  <string>水印检测</string>
              </property>
              <widget class="QLabel" name="label_4">
                  <property name="geometry">
                      <rect>
                          <x>65</x>
                          <y>30</y>
                          <width>31</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>阈值：</string>
                  </property>
              </widget>
              <widget class="QLineEdit" name="threshold_lineEdit">
                  <property name="geometry">
                      <rect>
                          <x>105</x>
                          <y>28</y>
                          <width>51</width>
                          <height>20</height>
                      </rect>
                  </property>
              </widget>
              <widget class="QLabel" name="label_5">
                  <property name="geometry">
                      <rect>
                          <x>160</x>
                          <y>30</y>
                          <width>16</width>
                          <height>16</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="textFormat">
                      <enum>Qt::AutoText</enum>
                  </property>
              </widget>
              <widget class="QPushButton" name="detect_pushButton">
                  <property name="geometry">
                      <rect>
                          <x>260</x>
                          <y>27</y>
                          <width>75</width>
                          <height>23</height>
                      </rect>
                  </property>
                  <property name="text">
                      <string>检测</string>
                  </property>
              </widget>
          </widget>
      </widget>
      <zorder>widget</zorder>
      <zorder>progressBar</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
